include/rmm/device_scalar.hpp

```diff
@@ -212,10 +231,9 @@ class device_scalar {
    */
   T const *data() const noexcept { return static_cast<T const *>(buffer.data()); }
 
-  device_scalar()                      = default;
-  ~device_scalar()                     = default;
-  device_scalar(device_scalar const &) = default;
```

Comment:
What happened to this one?

Reply:
This was moved up to include the optional stream and mr parameters.